lcls_tools/common/measurements/measurement.py

```diff
@@ -1,10 +1,23 @@
 from abc import ABC, abstractmethod
 from typing import Optional
-
 from pydantic import BaseModel, DirectoryPath
 
 
 class Measurement(BaseModel, ABC):
+    """
```

Comment:
Let's try the google style RadiaSoft recommended? https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html